src/adagrams.js

```diff
@@ -1,15 +1,133 @@
+const generatePoolArray = (LETTER_POOL) => {
+  let poolArray = [];
```

Comment:
```suggestion
  const poolArray = [];
```

Since `poolArray` isn't re-assigned a new value, we should use `const` instead. We can still update and add elements to a mutable collection like a list when we use `const`